modules/deploy/pages/deployment-option/self-hosted/manual/production/dev-deployment.adoc

```diff
@@ -7,13 +7,12 @@ You can deploy Redpanda using well-known configuration properties optimized for
 
 [NOTE]
 ====
-Development mode bypasses `fsync`, acknowledging messages before they're stored to disk. This reduces the durability of messages, could cause potential data loss, and could give unrealistic performance data for a production environment.
-
-See xref:./production-deployment.adoc[Deploy for Production] to deploy for a production environment.
-
-See xref:get-started:quick-start.adoc[Redpanda Quickstart] to try out Redpanda in Docker.
+* Development mode has write caching enabled by default, which lets Kafka clients produce to and consume data from topics without explicitly first flushing the data to disk. Write caching provides lower latency and higher throughput for testing, but it is done at the expense of durability guarantees. For more information, or to disable this, see xref:develop:produce-data/configure-producers.adoc#write-caching[write caching].
```

Comment:
"Write caching provides lower latency and higher throughput for testing"

@bharathv shouldn't we just say "lower latency, especially with slower storage media and with smaller batches" and forget the "higher throughput" and "for testing" parts?

If so we should make this change in all places where we describe this @micheleRP

Reply:
Yes, lets remove the "high throughput"  part for now. We don't yet have a conclusive evidence of higher throughput.

Also agree that we should remove "for testing" part gives an impression that it is only for testing.